data-serving/scripts/convert-data/parsers.py

```diff
@@ -239,6 +236,35 @@ def parse_location(value: Any) -> Dict[str, Any]:
     }
 
 
+def parse_location_list(value: Any) -> [Dict[str, Any]]:
+    '''
+    Parses one or more locations from a string.
+
+    Parameters:
+        value: Value representing a location or a delimiter-separated list of
+          locations.
+
+    Returns:
+      None: When the input value is empty or a string indicating n/a.
+      [Dict[str, Any]]: When the value is present and successfully parsed. 
+        A list of dictionaries containing the location data.
+    '''
+    separator = None
+    if pd.isna(value):
+        return None
+    if ';' in value:
+        separator = ';'
+    if ':' in value:
+        separator = ':'
+    if ' and ' in value:
```

Comment:
I'm not sure if there are many instances that'd trip this up, but there's [at least one](https://en.wikipedia.org/wiki/Rough_and_Ready,_California). :-P

I take it you've briefly sanity checked this separator list?

Reply:
eek... OK, let me remove this separator.

what i'm working on right now will throw an error if we can't match the location string, so if we end up having something like "NY and FL", this will get logged and we should fix it in the source to be "NY; FL"

similarly, I'm actually going to remove the check above for commas -- I'd really like us to standardize on a non-comma delimiter so commas can be used e.g. for "NY, New York, USA"